stock_warehouse_orderpoint_stock_info/views/stock_warehouse_orderpoint_view.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+        <record model="ir.ui.view" id="stock_warehouse_orderpoint_tree_view">
```

Comment:
> Place id attribute before model
please set id as first attribute

id should be view_warehouse_orderpoint_tree (same as core view id)